articles/container-apps/tutorial-event-driven-jobs.md

```diff
@@ -76,6 +90,32 @@ The job uses an Azure Storage queue to receive messages. In this section, you cr
         --connection-string "$QUEUE_CONNECTION_STRING"
     ```
 
+## Create a user-assigned managed identity
+
+You can pull images from private repositories in Microsoft Azure Container Registry using managed identities for authentication to avoid the use of administrative credentials. When possible, you should use a user-assigned managed identity to pull images.
```

Comment:
```suggestion
To avoid using administrative credentials, pull images from private repositories in Microsoft Azure Container Registry using managed identities for authentication. When possible, use a user-assigned managed identity to pull images.
```